From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Fri, 22 May 2020 18:28:47 -0400
Subject: Use pkg-config recommended by the environment

This should help with cross-building.  See suggestions from Helmut
Grohne on https://bugs.debian.org/958355

Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
---
 mk/defaults/linux.mk | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/mk/defaults/linux.mk b/mk/defaults/linux.mk
index 351c119..c84b83c 100644
--- a/mk/defaults/linux.mk
+++ b/mk/defaults/linux.mk
@@ -61,7 +61,9 @@ ifeq ($(LINUX_VARIANT),fedora)
 endif
 #(info USE_GLIBC_KERN_FLIP_HEADERS=$(USE_GLIBC_KERN_FLIP_HEADERS))
 
+PKG_CONFIG?=pkg-config
+
 ifndef NSS_CFLAGS
-  NSS_CFLAGS := $(shell pkg-config --cflags nss)
+  NSS_CFLAGS := $(shell $(PKG_CONFIG) --cflags nss)
   export NSS_CFLAGS
 endif
